docs/playbook/Module-0.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Module 0 - Getting started with Training
+description: "Training Module 0"
+layout: playbook_js
+tags: [playbook] [training]
+---
+
+# Module 0: Context and setup
+
+## Introduction
+### What is the goal of this module
+- In this module, you will learn about several tools you’ll need to interact with the Data platform infrastructure, and how to set them up.
+You will be able to find out which modules are best suited for your interests/goals
+- Find out what you need as prerequisites for the modules
+
+##  What is the goal of the Training as a whole
+<details>
+    <summary>Summary of what the purpose of this training is. Click to Expand </summary>
+
+The goal of the training is to understand the end-to-end journey of data in the platform and get to grips with the core elements and processes of the Data Platform, in a safe environment, using open data. Depending on what you want to do in the platform, you may not need to complete all the modules. However, the modules rely on each other and need to be completed in order. 
+After you complete the training, you will likely find that there is more you need to learn from the Playbook in order to use the Data Platform in your work (for example, performing other types of data transformation).
+
+### Module 1
+
+This module is intended for people who want to add/ingest data into the platform so they or others can use it in other tools (e.g. Qlik)
+After completing this module, you should be able to do the following:
+
+- Ingest Google Sheets into the Data Platform
+- Write a simple terraform module
+- Submit a pull request in Github
+- Navigate the AWS console
+- Run a job in AWS Glue
+- View data objects in AWS S3
+- Query data in AWS Athena
+### Module 2
+This module is intended for people who want to transform data within the Data Platform and make the result available for querying
+After completing this module, you should be able to do the following:
+- Load 1 or more Dataset from S3 into a coding environment, local and on AWS Glue
+- Learn how to test your Data Transformation in the Local Notebooking Environment ( saves money and is much faster to debug )
+- Learn how to do transformations in Spark syntax (including Joins)
+- Add a Glue job within the AWS console
+- Save the dataset after transformation into the refined zone
+
+### Module 3
+This module is intended for people who want to schedule or automate data transformations
+After completing this module, you should be able to do the following:
+- Deploy a Glue Job into the Data Platform Production environment, with scheduler to automate the process
+- Deploy a Glue job using Terraform
+</details>
+
+## Why we are moving away from old ways of work, and working using a Data Platform
```

Comment:
This reads a little weirdly, maybe "Why we are moving away from our old ways of working, and using a Data Platform" ?

Reply:
That makes sense yeah

Reply:
```suggestion
## Why we are moving away from old ways of work, and a Data Platform
```